style.css

```diff
@@ -0,0 +1,676 @@
+  font-size: 30px;
+  padding: 20px;
+  color: #000;
+}
+
+.nav-wrapper div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.nav-wrapper div > * {
+  text-decoration: none;
+  color: #272a31;
+  font-size: 40px;
+  margin: 30px;
+}
+
+.nav-wrapper a:hover {
+  color: #ec5242;
+  transition-duration: 0.5s;
+}
+
+li {
+  list-style: none;
+}
+
+.nav-wrapper-desktop {
+  display: none;
+}
+
+.nav-top-mobile {
+  display: flex;
+  position: relative;
+  background-color: #fff;
+  justify-content: center;
+  padding: 15px 0;
+}
+
+.nav-top-mobile img {
+  max-height: 51px;
+  cursor: pointer;
+}
+
+.nav-top-mobile button {
+  background: none;
+  border: none;
+  position: fixed;
+  font-size: 30px;
+  left: 3%;
+  top: 3%;
+  color: #272a31;
+  cursor: pointer;
+}
+
+.speakers button {
+  display: block;
+  background-color: transparent;
+  border: 1px solid #d3d3d3;
+  padding: 20px;
+  width: 90%;
+  cursor: pointer;
+}
+
+.main-program button {
+  background-color: #ec5242;
+  font-size: 16px;
+  padding: 30px 40px;
+  border-radius: 2px;
+  color: #fff;
+  border: none;
+}
+
+.nav-top-mobile button:hover {
+  color: #ec5242;
+}
+
+.presentation-container {
+  background-image: url(images/backgroundHeader.png);
+  display: flex;
+  flex-direction: column;
+  max-height: max-content;
+}
+
+.presentation-container h3 {
+  font-size: 50px;
+  font-weight: lighter;
+  color: #ec5242;
+  padding-top: 40px;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.presentation-container h1 {
+  font-size: 15vw;
+  font-weight: bolder;
+  background-clip: unset;
+  -webkit-background-clip: text;
+  color: transparent;
+  background-image: url(images/backgroundText.png);
+  padding-left: 30px;
+  max-width: 830px;
+}
+
+.main-description {
+  font-size: 13px;
+  color: #333;
+  background-color: #f7f7f8;
+  padding: 10px;
+  margin: 10px;
+  border: 4px solid #fff;
+}
+
+.presentation-container div {
+  display: flex;
+  flex-direction: column;
+  color: #333;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding-bottom: 100px;
+}
+
+.presentation-container h2 {
+  font-weight: bolder;
+  font-size: 30px;
+  margin-bottom: 10px;
+}
+
+.main-program {
+  background-image: url(images/patternBackground.png);
+  background-color: #272a31;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 70px;
+  padding-bottom: 70px;
+}
+
+.main-schedule {
+  margin: 30px 0;
+  width: 100%;
+}
+
+.main-program article {
+  display: flex;
+  padding: 10px;
+  max-width: 100%;
+  margin: 10px;
+  background-color: #97979738;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.main-program article:hover {
+  transform: scale(1.05);
+  box-shadow: 0 0 2px #ec5242;
+  transition-duration: 0.5s;
+}
+
+.main-program article > * {
+  margin: 10px;
+}
+
+.main-program h4 {
+  color: #ec5242;
+  font-size: 16px;
+  min-width: 72px;
+}
+
+.main-program p {
+  font-size: 12px;
+}
```

Comment:
- [ ] Your design is awesome 💯 However. I suggest using `text-align: center` here to center the paragraph to follow the original design layout 👌

> According to the requirements, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), Layout: composition and space between elements.

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/190312553-02f12cbf-d9a3-4130-87a4-95584b4a397c.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/190312579-e1790881-418f-4caa-9b78-bfa0ce2105b8.PNG" width="500" /> |